design/work_item_children.go

```diff
@@ -0,0 +1,23 @@
+package design
+
+import (
+	d "github.com/goadesign/goa/design"
+	a "github.com/goadesign/goa/design/apidsl"
+)
+
+var _ = a.Resource("work-item-children", func() {
```

Comment:
I need some clarity here. Is this question about the resource definition in the design package, if so I can move it from `work_item_children.go` to `work_item_link.go`. Please confirm.